README.rst

```diff
@@ -1,6 +1,84 @@
-==========================
- Transifex Python Library
-==========================
+Transifex Python Library
+========================
 
-The library is a wrapper around the Transifex API that tries to present
-a simpler interface to the developers.
+The library is a wrapper around the Transifex API, built to provide a
+simpler interface to users of the API.
+
+At this moment, it supports only a subset of the endpoints of the API.
+In particular, it allows certain operations on Projects, Resources and
+Translations.
+
+Usage
+-----
+
+Initialization
+~~~~~~~~~~~~~~
+
+Before making any requests, you need to setup the HTTP handler, which
+will be used for all subsequent requests.
+
+Authentication works both with username/password or with an API token.
+
+\```python from txlib.http.auth import BasicAuth from
```

Comment:
Why did you escape the backticks? There is something wrong with the newlines here.